packages/fxa-settings/src/pages/Signup/ConfirmSignupCode/en.ftl

```diff
@@ -2,6 +2,9 @@
 ## Users see this page after they have initiated account sign up,
 # and a confirmation code has been sent to their email address.
 
+# Page title show in browser title bar or page tab
+confirm-signup-code-page-title = Enter confirmation code
```

Comment:
Don't see this ID used anywhere, but I _did_ find a suspicious `-psignup-`, which _might_ be a typo.

```js
  // packages/fxa-settings/src/pages/Signup/ConfirmSignupCode/index.tsx#L208-211
  const localizedPageTitle = ftlMsgResolver.getMsg(
    'confirm-psignup-code-page-title',
    'Enter confirmation code'
  );
```

Reply:
Seems I need to update my prescription glasses. Thanks for catching these typos!!